styles.css

```
@@ -1,4 +1,4 @@
-@import url('https://fonts.googleapis.com/css2?family=Zen+Dots&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Zen+Dots&family=Share+Tech+Mono&display=swap');
```

Stylelint (17.5.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Zen+Dots&family=Share+Tech+Mono&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Zen+Dots&family=Share+Tech+Mono&display=swap'" (import-notation)

(import-notation)